src/api/app/helpers/webui/user_helper.rb

```diff
@@ -12,6 +12,16 @@ def user_image_tag(user, opt = {})
     image_tag(url, width: size, height: size, alt: alt, class: opt[:css_class])
   end
 
+  def webui2_user_image_tag(user, css_class = nil, alt = nil)
+    alt ||= (user ? "#{user.name}'s avatar" : 'Avatar')
+    url = if user.try(:email) && ::Configuration.gravatar
+            "https://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(user.email.downcase)}?s=500&d=wavatar"
+          else
+            'default_face.png'
+          end
+    image_tag(url, alt: alt, class: css_class, width: '100%')
+  end
```

Comment:
How about this:

* We put the `image_tag(...)` to the view
* Define a `gravatar_url` and `gravatar_tooltip` and use them in the `image_tag` call

Reply:
why?